amanuensis/blueprints/project.py

```diff
@@ -12,16 +12,39 @@
 from amanuensis.schema import ProjectSchema
 from cdislogging import get_logger
 
-
 blueprint = flask.Blueprint("projects", __name__)
 
 logger = get_logger(__name__)
 
+def get_state_id_to_code():
+
+    with flask.current_app.db.session as session:
+
+        # Map the State.id to the State.code in case they're in a different order than expected in the DB.
+
+       return {int(state.id): state.code for state in session.query(State).all()}
+
 # cache = SimpleCache()
 
 
+def determine_status_code(codes):
+    code = None
+    if "REJECTED" in codes:
+        code = "REJECTED"
+    elif "DATA_DELIVERED" in codes:
+        code = "DATA_DELIVERED"
+    elif "IN_REVIEW" in codes:
+        code = "IN_REVIEW"
+    elif "APPROVED" in codes:
+        code = "APPROVED"
+    else:
+        code = "UNKNOWN"
+    return code
```

Comment:
This works for now, but we should create a ticket in Jira to handle this on a DB level or config level instead of hardcoded since it will be different from consortia to consortia.

Reply:
Ok. I'll go ahead an create a ticket.